client/src/index.jade

```diff
@@ -0,0 +1,48 @@
+doctype html
+html(lang="en")
+  head
+    meta(charset="utf-8")
+    meta(name="viewport" content="width=device-width, initial-scale=1")
+    title Situation Room
+    link(rel="stylesheet" href="css/main.css")
+  body
+    #app-content
+    script(type='text/template', id="homeTemplate").
+      Situation Room - like <a href="https://www.pingdom.com/">pingdom</a>, but with more cachet.
+      <p><a href="#create">Enter New Website</a></p>
+      <p><a href="#list_sites">Show Sites</a></p>
+    script(type='text/template', id="formTemplate").
+      <form>
+      <table>
+        <legend>Enter a new site URL to check.</legend>
+        <tr><td><label>URL: </label></td><td><input type='url' name='url' placeholder='URL'></tr></td>
+        <tr><td><label>Email: </label></td><td><input type='email' name='email' placeholder='Email'></tr></td>
+        <tr><td><button type='submit'>Create</button></tr></td>
+      </table>
+      </form>
+      <p><a href='/'>Home</a></p>
+    script(type='text/template', id="listTemplate").
+      <table id="site-table">
+        <thead>
+          <tr>
+            <th>Key</th>
+            <th>Site URL</th>
+            <th>Status</th>
+            <th>Last Checked</th>
+          </tr>
+        </thead>
+        <tbody>
+        </tbody>
+      </table>
+      <p><a href="/">Home</a></p>
+    script(type='text/template', id="siteTemplate").
+      <tr>
+        <td><%= key %></td>
+        <td><%= url %></td>
+        <td><%= status_code %></td>
+        <td><%= last_checked_at %></td>
+      </tr>
+    script(type='text/template', id="noSitesTemplate").
+      <span>No sites listed.</span>
+      <p><a href="/">Home</a></p>
+    script(type='text/javascript', src="js/main.js")
```

Comment:
Since you're using webpack, you can remove the need for script templates by using [jade-loader](https://github.com/webpack/jade-loader).